migrator/migrations/sqls/20200220183445-seed-up.sql

```diff
@@ -0,0 +1,78 @@
+CREATE EXTENSION IF NOT EXISTS "uuid-ossp";
+
+CREATE TYPE reg_state AS ENUM ('unregistered', 'registered');
+
+CREATE TABLE IF NOT EXISTS Users (
+    uid uuid DEFAULT uuid_generate_v1() UNIQUE,
+    email varchar(255) NOT NULL,
+    password varchar(255) NOT NULL,
+    first_name varchar(255) NOT NULL,
+    last_name varchar(255) NOT NULL,
+    organizer boolean DEFAULT FALSE,
+    PRIMARY KEY (uid)
+);
+
+
+CREATE TABLE IF NOT EXISTS Hackathons (
+    hid uuid DEFAULT uuid_generate_v1() UNIQUE,
+    name varchar(255) NOT NULL,
+    start_date timestamp NOT NULL,
+    end_date timestamp NOT NULL,
+    location varchar(255) NOT NULL,
+    max_reg smallint,
+    draft boolean DEFAULT FALSE,
```

Comment:
Should this be false by default, or true (as in "this is a draft until you switch it on")? I don't know if this matters much because you'll probably be doing things explicitly on the server side anyways!

Reply:
Good point. I think It should default to `true` actually.